android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/ShopifyUnityPlayerActivity.java

```diff
@@ -14,16 +14,18 @@
 import com.shopify.unity.buy.androidpay.GoogleApiClientFactory;
 import com.shopify.unity.buy.androidpay.view.viewmodel.ConfirmationViewModel;
 import com.shopify.unity.buy.androidpay.view.widget.ConfirmationView;
+import com.shopify.unity.buy.androidpay.view.widget.ShippingMethodSelectDialog;
 import com.shopify.unity.buy.androidpay.view.widget.WalletFragmentInstaller;
+import com.shopify.unity.buy.models.CheckoutInfo;
 import com.shopify.unity.buy.models.ShippingMethod;
 import com.unity3d.player.UnityPlayerActivity;
 
-import java.util.List;
+import static com.shopify.unity.buy.androidpay.view.widget.ShippingMethodSelectDialog.OnShippingMethodSelectListener;
```

Comment:
Nit: Could you just import the package instead of statically importing the member?